docs/mine/lotus/manage-storage-deals.md

```diff
@@ -1,33 +1,139 @@
 ---
 title: 'Lotus Miner: manage storage deals'
-description: 'This guide describes the different workflows and options to effectively manage storage deal conditions requested set by Lotus Miners.'
+description: 'This guide describes the different workflows and options that Lotus Miners can use to manage storage deals.'
 breadcrumb: 'Manage storage deals'
 ---
 
 # {{ $frontmatter.title }}
 
 {{ $frontmatter.description }}
 
-::: callout
-This page is a _stub_! We are working on expanding these docs. If you have ideas, please contribute to improving this page.
-:::
+During the lifetime of a miner, Filecoin network clients, will query the storage price-ask advertised by the miner and initiate deals. Deals go through several phases which, at high level are:
 
-## Exploring deals
+1. Data transfer (for online deals) or data import (for offline deals)
+2. Sealing sector with deal data (miner)
+3. Proving (every 24 hours)
 
-While the Lotus Miner is running, the `lotus-miner` application can be used to manage and configure some aspects of the deals like the storage-ask details. The following command shows the self-documented commands around storage deals:
+The following sections provides insights into the different ways Lotus can be used to manage several parts of the storage-deal process.
+
+## Enabling and disabling deals
+
+There are two ways to enable and disable new storage deals in the miner:
+
+- Editing the `[DealMaking]` options in the [miner configuration file](miner-configuration.md) and [restarting the miner](miner-lifecycle.md). Since re-starting the miner is a delicate operation, it is better to let Lotus handle it by:
+- Using the `lotus-miner storage-deals selection` commands.
```

Comment:
Are these two bullet-points the two ways to enable and disable deals? If so, it'd be better if we wrote something like:

```suggestion
There are two ways to enable and disable new storage deals in the miner. Either:

- Edit the `[DealMaking]` options in the [miner configuration file](miner-configuration.md) and [restarting the miner](miner-lifecycle.md).
- Using the `lotus-miner storage-deals selection` commands.

Since restarting the miner is a delicate operation, it is best to let Lotus handle things by using the `lotus-miner storage-deals selection` commands.
```